common.js

```diff
@@ -138,6 +138,13 @@ var vcwg = {
       authors: ['Orie Steele', 'Michael Jones'],
       status: 'ED',
       publisher: 'W3C Verifiable Credentials Working Group'
+    },
+    'VC-SPECS-DIR': {
+      title: 'Verifiable Credentials Specifications Directory',
+      href: 'https://w3c.github.io/vc-specs-dir/',
```

Comment:
I think this changes to a TR link when this becomes a note, right?

Reply:
> I think this changes to a TR link when this becomes a note, right?

Yes, correct.